src/edu/psu/sweng500/team8/gui/BoardGUI.java

```diff
@@ -0,0 +1,261 @@
+package edu.psu.sweng500.team8.gui;
+
+import java.awt.Dimension;
+import java.awt.GridBagConstraints;
+import java.awt.event.FocusAdapter;
+import java.awt.event.FocusEvent;
+import java.awt.event.MouseAdapter;
+import java.awt.event.MouseEvent;
+import java.util.HashSet;
+import java.util.Set;
+
+import javax.swing.JLabel;
+import javax.swing.JPanel;
+import javax.swing.JTextField;
+
+import edu.psu.sweng500.team8.coreDataStructures.Cell;
+import edu.psu.sweng500.team8.coreDataStructures.CellCoordinates;
+import edu.psu.sweng500.team8.play.GameSession;
+
+public class BoardGUI extends JPanel {
+	
+	private static final int BOARD_SIZE = 486;
+	private BlockGUI[][] blocks;
+	private GameSession gameSession;
+	private CellGUI selectedCell;
+	private Set<CellGUI> highlightedIncorrectCells = new HashSet<CellGUI>();
+	
+	public BoardGUI() {
+		
+		setPreferredSize(new Dimension(BOARD_SIZE, BOARD_SIZE));
+		setMaximumSize(new Dimension(BOARD_SIZE, BOARD_SIZE));
+		setMinimumSize(new Dimension(BOARD_SIZE, BOARD_SIZE));
+
+		blocks = new BlockGUI[3][3];

```

Comment:
If you read the whole thing, it's not required in case there is no local
variable with the same name. But I will still update them as much as
possible.

On Sun, Jul 12, 2015 at 10:07 PM, Jeremy Nelson <notifications@github.com>
wrote:

> In src/edu/psu/sweng500/team8/gui/BoardGUI.java
> <https://github.com/garnryang/Team8/pull/13#discussion_r34429516>:
>
> > +
> > +public class BoardGUI extends JPanel {
> > +	
> > +	private static final int BOARD_SIZE = 486;
> > +	private BlockGUI[][] blocks;
> > +	private GameSession gameSession;
> > +	private CellGUI selectedCell;
> > +	private Set<CellGUI> highlightedIncorrectCells = new HashSet<CellGUI>();
> > +	
> > +	public BoardGUI() {
> > +		
> > +		setPreferredSize(new Dimension(BOARD_SIZE, BOARD_SIZE));
> > +		setMaximumSize(new Dimension(BOARD_SIZE, BOARD_SIZE));
> > +		setMinimumSize(new Dimension(BOARD_SIZE, BOARD_SIZE));
> > +
> > +		blocks = new BlockGUI[3][3];
>
> Java style (from what I gather) is to use "this.variable" to differentiate
> member variables from local variables.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/garnryang/Team8/pull/13/files#r34429516>.
>